packages/components/src/components/teaser/teaser.ts

```diff
@@ -107,7 +107,11 @@ export default class SdTeaser extends SolidElement {
       >
         <div
           style=${this._orientation === 'horizontal' ? `width: var(--distribution-media, 100%);` : ''}
-          class=${cx(!inset && this._orientation === 'vertical' && 'mb-4', !slots['teaser-has-media'] && 'hidden')}
+          class=${cx(
+            !inset && this._orientation === 'vertical' && 'mb-4',
+            !slots['teaser-has-media'] && 'hidden',
+            this.variant === 'white border-neutral-400' && this._orientation === 'vertical' && '-m-px'
```

Comment:
The solution works great. I considered utilizing arbitrary values in this context though.

```suggestion
            this.variant === 'white border-neutral-400' && this._orientation === 'vertical' && 'm-[-1px]'
```

Reply:
Why? Arbitrary values are last resort and should be avoided. If someone else uses -m-px somewhere else (what would be correct!) we have two classes for the same thing, what bloats our CSS.

Reply:
Ah. Now I get the problem. We didn't hab"ve the class. :D What's a problem of our spacing scale... let's keep that in mind. We should add that to our scale.

Reply:
Created a ticket for it here -> [#664](https://github.com/solid-design-system/solid/issues/664)